de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/migrator/AMigrator.java

```diff
@@ -204,6 +222,34 @@ protected void removeFromTarget(ReferenceChange diff, boolean rightToLeft) {
 	protected void registerOldToNewIds() {
 	}
 	
+	@Override
+	public Set<String> getNewDependencies(Concept concept, IMigrator previousMigrator) {
+		
+		String conceptId = concept.getFullQualifiedName() + "/";
+		Concept conceptPrevious = loadConceptXmi(conceptId + previousMigrator.getResource());

```

Comment:
According to ther javadoc, `previousMigrator` can be null, which would result in NPE here

Reply:
Oh true, and actually the same holds for the migration itself